modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/spring/EagerObjectCreator.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.spring.dsl.spring;
```

Comment:
most of the spring classes have to be moved to internal. I will do it in the next PR for spring module that is scheduled for next sprint